core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -2132,6 +2174,7 @@ public boolean rename(TachyonURI srcPath, TachyonURI dstPath) throws FileDoesNot
     synchronized (mRootLock) {
       Inode inode = getInode(srcPath);
       if (inode == null) {
+        mMasterSource.incRenameOps();
```

Comment:
Because EditLog.loadSingleLog also calls renameInternal and other xxInternal methods, I increment the counter in the renameInternal method.